### PR TITLE
build: package wheels for cp38 and cp39

### PR DESCRIPTION
**Pull request by rr-**

### CODE REVIEW

File: .github/workflows/deploy-binaries.yml [hunk 1]
```diff
@@ -270,7 +270,7 @@ jobs:
         PKG="ffms2.zip"
         curl -LO "$LINK/$PKG"
         unzip $PKG
-        python setup.py bdist_wheel --plat-name win_amd64 --python-tag cp37
+        python setup.py bdist_wheel --plat-name win_amd64 --python-tag cp37 --python-tag cp38 --python-tag cp39
```
Comment (rr-) [inline .github/workflows/deploy-binaries.yml:273]: I'm not sure if this is the way to specify multiply tags or should I use a comma-separated list of values. I didn't find any documentation about it.

Comment (Luni-4) [reply .github/workflows/deploy-binaries.yml:273]: IIRC we need to write tags like this: `--python-tag cp37-cp38-cp39`

Comment (Luni-4) [reply .github/workflows/deploy-binaries.yml:273]: Another solution: https://stackoverflow.com/a/52613394

Comment (Luni-4) [reply .github/workflows/deploy-binaries.yml:273]: @rr- 

Remove temporary the code to publish on `PyPI` from CI and try the solution above, in this way you can see if it works correctly.

**Comment by rr-** [comment]
@Luni-4 it seems to have worked?

**Comment by rr-** [comment]
Actually I don't think it has worked - `wheel2json` shows that the tag is only the last `--python-tag`. Your suggestion to use `cp37-cp38-cp39` also doesn't seem to be working - `wheel2json` displays that the created wheel has invalid filename. I found this:
https://www.python.org/dev/peps/pep-0425/#compressed-tag-sets
>A single-source Python 2/3 compatible distribution can use the compound tag py2.py3.
Using `--python-tag cp37.cp38.cp39` seems to be working. I'll commit it again in a minute.

**Comment by Luni-4** [comment]
@rr- 

This is good, it seems to work now. We can remove the test commit and we are ready to go

**Pull request merged**